index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Contact us page</title>
+</head>
+<body>
+    <section>
+        
+    <div class="container">
+        <div class="contactinfo">
+          <div>
+           <h2>Contact info</h2> 
+            <ul class="info">
+                <li>
+                    <span>
+                        <img src="img1/kindpng_1084606.png" width="112px" alt="location"></span>
+                        <span>202 Grant raod<br>
+                            California, CA<br>
+                            396191
+                        </span>
+                </li>
+                <li>
+                    <span>
+                        <img src="img1/email-13762.png" width="112px" alt="location"></span>
+                        <span>helloworld@gmail.com</span>
+                </li>
+                <li>
+                    <span>
+                        <img src="img1/icons8-phone-512.png" width="112px" alt="location"></span>
+                        <span>310-354-903 </span>
+                </li>
+            </ul>
+            <ul class="sci">
+                <li><a href="#"><img src="img1/1.png" width="40px"></a></li>
```

Comment:
Your icon image looks a bit pixelated. please use icons from websites like favicon, icons8, there are many options to choose from.

Reply:
ok, thanks for suggestion